src/nft/components/bag/profile/ListingButton.tsx

```diff
@@ -211,7 +211,7 @@ export const ListingButton = ({ onClick, buttonText, showWarningOverride = false
 
   const warningWrappedClick = () => {
     if ((!disableListButton && canContinue) || showWarningOverride) {
-      if (issues && isNftListV2 && !showResolveIssues) toggleShowResolveIssues()
+      if (issues && isNftListV2) !showResolveIssues && toggleShowResolveIssues()
```

Comment:
This was actually correct. I basically want this click to do nothing if there is issues and we're already showing the 'Resolve Issues' text. Including`showResolveIssues` in the check triggered the onClick function when we wanted it to do nothing